lib/user/mocks/user-client-mock.js

```diff
@@ -0,0 +1,24 @@
+var sinon = require('sinon');
```

Comment:
@nialldonnellyfh yep just wanted to discuss if we should put -spec and -mock into same path with modules or move them to different directories. I prefer to have in same directory where functional module is located but it depends on agreement